assets/src/edit-story/components/helpCenter/toggle/index.js

```diff
@@ -46,31 +47,30 @@ const Button = styled(dsButton)`
 `;
 
 const Label = styled.span`
-  display: none;
-
-  @media ${({ theme }) => theme.breakpoint.desktop} {
-    display: block;
-    line-height: 20px;
-    text-align: left;
-
-    & ~ div {
-      margin-left: 20px;
-    }
-  }
+  display: block;
+  line-height: 20px;
+  text-align: left;
 `;
 
 const NotificationWrapper = styled.div`
-  margin: -2px 0 -2px 8px;
+  margin: -2px 0 -2px 20px;
 `;
 
-const HelpIcon = styled(Icons.QuestionMarkOutline)`
+const Chevron = styled(Icons.ChevronUpSmall)`
   display: block;
-  margin: -6px 0;
+  margin: -6px 0 -6px 16px;
   width: 32px;
   height: 32px;
```

Comment:
Probably can remove these right since each icon is already 32x32?
```suggestion
```

Reply:
We strip the width and height property from the `<svg />` element in build via `svgr`, so without this, the icon just scales up to fit their container. And for the large square button, that's the full 56x56px.